website/pages/en/substreams/[[...slug]].mdx

```diff
@@ -22,20 +22,32 @@ export async function getStaticProps({ params: { slug = ['README'] } }) {
   if (response.status === 404 && paths !== 'README') {
     response = await fetch(`${baseURL}${paths}/README.md`)
   }
-  const data = await response.text()
+  const data = (await response.text())
+    // replace {% embed ... %} with <iframe />
+    .replaceAll(
+      /{%\s+embed\s+url="(.*?)"\s+%}/g,
+      (...m) =>
+        `<iframe src="${m[1].replace(
+          // we need enhance YouTube links, otherwise they will be not loaded in iframe
+          'youtube.com/watch?v=',
+          'youtube.com/embed/'
+        )}" style={{aspectRatio: 16/9, width: '100%'}}/>`
+    )
```

Comment:
this fix 

![image](https://user-images.githubusercontent.com/7361780/236554041-11139e7e-57fb-4eba-bd7a-35d5130694e9.png)

---

![image](https://user-images.githubusercontent.com/7361780/236554051-4dfaf4b2-25f1-4db4-a266-55d459d685a6.png)